src/lib/Interface.js

```diff
@@ -270,7 +278,11 @@ export default class Interface {
         return;
       }
 
-      that.render('style', that.buildStyle());
+      if (window.CookieConsent.config.customCSS) {
+        that.injectCustomStyles(window.CookieConsent.config.customCSS);
+      } else {
+        that.render('style', that.buildStyle());
+      }
```

Comment:
Shouldn't we have the ability to keep the original css, and add some small changes?
I would add another config option to ignore built in css, and not ignore it all when custom css is added.